script.js

```diff
@@ -0,0 +1,199 @@
+const latesticonmobile1 = document.querySelector('#latesticonmobile');
+
+coursesiconmobile1.addEventListener('click', () => {
+  menubarfluid.classList.remove('menu-bar-fluid-background-change');
+  navlistmobile.classList.remove('nav-list-mobile-change');
+  wrap.classList.remove('change-wrap');
+});
+joiniconmobile1.addEventListener('click', () => {
+  menubarfluid.classList.remove('menu-bar-fluid-background-change');
+  navlistmobile.classList.remove('nav-list-mobile-change');
+  wrap.classList.remove('change-wrap');
+});
+sponsericonmobile1.addEventListener('click', () => {
+  menubarfluid.classList.remove('menu-bar-fluid-background-change');
+  navlistmobile.classList.remove('nav-list-mobile-change');
+  wrap.classList.remove('change-wrap');
+});
+latesticonmobile1.addEventListener('click', () => {
+  menubarfluid.classList.remove('menu-bar-fluid-background-change');
+  navlistmobile.classList.remove('nav-list-mobile-change');
+  wrap.classList.remove('change-wrap');
+});
+
+const lecturesData = [
+  {
+    lecturerName: 'Yochai Benkler',
+    lecturerCertification: 'Full stack Developer Microverse Certified',
+    lecturerInfo: 'Benkler has Contributed in Many Open-Source Softwares and is a well known Europe Professor',
+    lecturerImage: '/images/lecturer1.jpg',
+  },
+  {
+    lecturerName: 'Kilnam Chon',
+    lecturerCertification: 'Full stack Developer Microverse Certified',
+    lecturerInfo: 'Kilnam has Contributed in Many Open-Source Softwares and is a well known Harverd Professor',
+    lecturerImage: '/images/lecturer2.jpg',
+  },
+  {
+    lecturerName: 'SoYeong Noh',
+    lecturerCertification: 'Full stack Developer Microverse Certified',
+    lecturerInfo: 'SoYeong has Contributed in Many Open-Source Softwares and is a well known Harverd Professor',
+    lecturerImage: '/images/lecturer3.jpg',
+  },
+  {
+    lecturerName: 'Julia Leda',
+    lecturerCertification: 'Full stack Developer Microverse Certified',
+    lecturerInfo: 'Julia has Contributed in Many Open-Source Softwares and is a well known Harverd Professor',
+    lecturerImage: '/images/lecturer4.jpg',
+  },
+  {
+    lecturerName: 'Lila Tretikov',
+    lecturerCertification: 'Full stack Developer Microverse Certified',
+    lecturerInfo: 'Lila has Contributed in Many Open-Source Softwares and is a well known Harverd Professor',
+    lecturerImage: '/images/lecturer5.jpg',
+  },
+  {
+    lecturerName: 'Reyan Merkley',
+    lecturerCertification: 'Full stack Developer Microverse Certified',
+    lecturerInfo: 'Ryan has Contributed in Many Open-Source Softwares and is a well known Harverd Professor',
+    lecturerImage: '/images/lecturer6.jpg',
+  },
+];
+
+lecturesData.forEach(() => {
+  featuresection.innerHTML = `
+    <div class="container feature-program-container ">
+    <div class="row">
+        <div class="col-12 d-flex justify-content-center feature-main-text">Featured Speakers</div>
+    </div>
```

Comment:
- [ ] Please this part of your project is not responsive to a media screen of width `375px` so make sure you adjust the margin between `Featured Speakers` text and `Enroll Now in Course Batch 2023` button to look according to the [given design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015).
![image](https://user-images.githubusercontent.com/92303891/218155281-a2c3ab65-9039-4134-90ef-3bedfacefb03.png)

Reply:
![image](https://user-images.githubusercontent.com/116083031/218163357-844068e3-4ccf-4409-ba3c-af828900b24f.png)
Please kindly see its responsive as well